it/uk/gov/hmrc/helptosave/MissingMandatoryData.scala

```diff
@@ -0,0 +1,175 @@
+package uk.gov.hmrc.helptosave
+
+import java.net.URLEncoder
+import java.time.LocalDate
+import java.util.Base64
+
+import com.github.tomakehurst.wiremock.client.WireMock._
+import org.scalatest.{BeforeAndAfterAll, BeforeAndAfterEach, Matchers, WordSpec}
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.time.{Millis, Seconds, Span}
+import org.scalatestplus.play.guice.GuiceOneServerPerSuite
+import play.api.http.Status
+import play.api.http.Status._
+import play.api.libs.json.Json
+import play.api.libs.ws.{WSClient, WSResponse}
+import uk.gov.hmrc.helptosave.connectors.CitizenDetailsConnector.{CitizenDetailsAddress, CitizenDetailsPerson, CitizenDetailsResponse}
+import uk.gov.hmrc.helptosave.connectors.UserDetailsConnector.UserDetailsResponse
+import uk.gov.hmrc.helptosave.models.MissingUserInfo.{Email, Surname}
+import uk.gov.hmrc.helptosave.models.{Address, UserInfo}
+import uk.gov.hmrc.helptosave.services.UserInfoService.UserInfoServiceError.MissingUserInfos
+import uk.gov.hmrc.helptosave.support.FakeRelationshipService
+import uk.gov.hmrc.helptosave.util.NINO
+
+
+class MissingMandatoryData extends WordSpec
```

Comment:
The NSIUserInfo model in the frontend doesnt contain any options, so we cannot test missing info scenarios in the frontend